API/models/reviewDao.js

```diff
@@ -0,0 +1,24 @@
+const appDataSource = require('../models/appDataSource');
+const { DatabaseError } = require('../utils/error');
+
+const reviewUpdate = async (userId, productId, title, content, url) => {
+  try {
+    const aa = await appDataSource.query(
+      `UPDATE reviews
+        LEFT JOIN review_images ON (reviews.id = review_images.review_id)
+          SET 
+            reviews.title = "${title}",
```

Comment:
```suggestion
            reviews.title = ?,
```
변수를 직접 넣지 말고 ?을 사용해서 넣어주세요.